docs/docs/interpreter/ignite.md

```diff
@@ -0,0 +1,98 @@
+
+ <table class="table-configuration">
+  <tr>
+      <th>Property Name</th>
+      <th>value</th>
+      <th>Description</th>
+  </tr>
+  <tr>
+      <td>ignite.addresses</td>
+      <td>127.0.0.1:47500..47509</td>
+      <td>Coma separated list of Ignite cluster hosts. See [Ignite Cluster Configuration](https://apacheignite.readme.io/v1.2/docs/cluster-config) section for more details.</td>
+  </tr>
+  <tr>
+      <td>ignite.clientMode</td>
+      <td>true</td>
+      <td>You can connect to the Ignite cluster as client or server node. See [Ignite Clients vs. Servers](https://apacheignite.readme.io/v1.2/docs/clients-vs-servers) section for details. Use true or false values in order to connect in client or server mode respectively.</td>
+  </tr>
+  <tr>
+      <td>ignite.config.url</td>
+      <td></td>
+      <td>Configuration URL. Overrides all other settings.</td>
+   </tr
+   <tr>
+      <td>ignite.jdbc.url</td>
+      <td>jdbc:ignite:cfg://default-ignite-jdbc.xml</td>
+      <td>Ignite JDBC connection URL.</td>
+   </tr>
+   <tr>
+      <td>ignite.peerClassLoadingEnabled</td>
+      <td>true</td>
+      <td>Enables peer-class-loading. See [Zero Deployment](https://apacheignite.readme.io/v1.2/docs/zero-deployment) section for details. Use true or false values in order to enable or disable P2P class loading respectively.</td>
+  </tr>
+ </table>
+
+![Configuration of Ignite Interpreter](https://github.com/AhyoungRyu/Interpreter-Document/blob/master/Ignite%20Img/IgniteInterpreterConf.png?raw=true)
+
+### Interpreter Binding for Zeppelin Notebook
+After configuring Ignite interpreter, create your own notebook. Then you can bind interpreters like below image.
+
+![Binding Interpreters](https://github.com/AhyoungRyu/Interpreter-Document/blob/master/Ignite%20Img/IgniteInterpreterBinding.png?raw=true)
+
+For more interpreter binding information see [here](http://zeppelin.incubator.apache.org/docs/manual/interpreters.html).
+
+### How to use Ignite SQL interpreter
+In order to execute SQL query, use ` %ignite.ignitesql ` prefix. <br>
+Supposing you are running `org.apache.ignite.examples.streaming.wordcount.StreamWords`, then you can use "words" cache(Of course you have to specify this cache name to the Ignite interpreter setting section `ignite.jdbc.url` of Zeppelin). 
+For example, you can select top 10 words in the words cache using the following query
+
+  ``` 
+  %ignite.ignitesql 
+  select _val, count(_val) as cnt from String group by _val order by cnt desc limit 10 
+  ``` 
+  
+  ![IgniteSql on Zeppelin](https://github.com/AhyoungRyu/Interpreter-Document/blob/master/Ignite%20Img/IgniteSqlExample.png?raw=true)
+  
+As long as your Ignite version and Zeppelin Ignite version is same, you can also use scala code. Please check the Zeppelin Ignite version before you download your own Ignite. 
+  
+  ![Using Scala Code](https://github.com/AhyoungRyu/Interpreter-Document/blob/master/Ignite%20Img/ScalaCode.png?raw=true)
```

Comment:
is it possible to include the full source code here or link to it? It's much easier for someone to copy/paste into their notebook to try it out if the text is available